.github/scripts/run-container-test.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Run precompiled Kotlin/Native test binaries in a Docker container for a specific Linux distribution and architecture.
+
+This requires Docker multiarch support, see https://docs.docker.com/build/building/multi-platform/ and https://github.com/multiarch/qemu-user-static
+In GitHub we use a provided action for this: https://github.com/docker/setup-qemu-action
+
+Locally you would need to run one of:
+
+`docker run --rm --privileged multiarch/qemu-user-static --reset -p yes --credential yes`
+
+OR
```

Comment:
question: how do we know to run one or the other?

Reply:
I don't know the answer to that but locally I used the `multiarch/qemu-user-static` but the GH action from Docker uses the other. I haven't dived into the differences.